### PR TITLE
Make dialog box smaller and remove underlines on home-page icons

### PR DESCRIPTION
**Pull request by Haider8**

### CODE REVIEW

File: src/components/Home.vue [hunk 1]
```diff
@@ -243,4 +243,7 @@ export default {
 .news-item {
   font-size: 16px;
 }
+a {
+  text-decoration: none;
+}
```
Comment (iamareebjamal) [inline src/components/Home.vue]: Only match icon links, this way all links will have no underlines

File: src/components/Other/Projects.vue [hunk 2]
```diff
@@ -48,8 +48,7 @@
       <div class="text-xs-center">
         <v-dialog
           v-model="dialog"
-          full-width
-          scrollable
+          width="500"
```
Comment (iamareebjamal) [inline src/components/Other/Projects.vue]: Use max-width instead

Comment (Haider8) [reply src/components/Other/Projects.vue]: max-width will increase the size of the dialog box

Comment (iamareebjamal) [reply src/components/Other/Projects.vue]: OK

**Comment by iamareebjamal** [comment]
Check #136

**Pull request merged**